packages/web-app-files/src/components/Upload/DetailsWidget.vue

```diff
@@ -5,8 +5,12 @@
         <oc-icon name="file_copy" class="oc-mr-s" />
         <div class="uk-width-expand">
           <div class="uk-flex">
-            <div class="oc-text-bold uk-width-expand uk-text-truncate">{{ item.name }}</div>
-            <div class="uk-width-auto uk-text-nowrap">{{ getResourceSize(item.size) }}</div>
+            <div id="item-name" class="oc-text-bold uk-width-expand uk-text-truncate">
```

Comment:
Since the `id` has to be unique for the entire rendered page, please always include some sort of namespacing in ids. Usually we have something like the app name (`files`) and the component (`upload-details`). You could choose `files-upload-details-item-name` or something similar.